### DISCUSSION TITLE
What happens to a release done on a forked repository if that forked repository is deleted?

### DISCUSSION THREAD

**Discussion by ijcardin** [community/community#142092] [Repositories] [answered]
### Select Topic Area

Question

### Body

Sorry if this is a silly question.

I have forked a repository and added my own changes to it and created a release. This was then used as a dependency in my own project. When I deleted this forked repository (as it was no longer needed) I found that Gradle was still able to find and use this release, even though the repository has been deleted. Is this normal?

**Comment by ynng3** [comment, marked as answer]
When a forked repository is deleted, the releases made in that forked repository are also deleted. However, if Gradle or another dependency management tool can still find and use the release, it likely means that the release has already been downloaded and cached locally, allowing it to be used even after the repository has been deleted.
